### DISCUSSION TITLE
How do I get the "<-" character from the keyboard?

### DISCUSSION THREAD

**Discussion by neleabels** [dirkwhoffmann/virtualc64#758] [Q&A]
I am using a German keyboard and have set the emulator to symbolic mapping. That way, I can get nearly every important character - with the exception of the back arrow ("<-") 

I can get around it with the virtual keyboard and the menu, but this would be a pity since it breaks the usability of Turbo Macro Pro.

Am I missing something or is there a key combination which I have not found?

**Comment by neleabels** [comment]
Weird. Now the "ü" key accurately returns the back arrow. I don't know what I did wrong.

**Comment by dirkwhoffmann** [comment]
The left arrow key is mapped to `ü` and `PAGE DOWN`. 

You'll find the complete list in source file `KeyboardController.swift`. E.g., this is where `ü` is mapped:

```Swift
extension C64Key {
    
    /* Translates a character to a list of C64 keys. This function is called in
     * symbolic key mapping mode when the user hits a key.
     */
    static func translate(char: String?) -> [C64Key] {
        
        if char == nil { return [] }
        
        switch char! {
            
        // First physical row
        case "ü": return [C64Key.leftArrow]
        case "1": return [C64Key.digit1]
        case "!": return [C64Key.digit1, C64Key.shift]
        case "2": return [C64Key.digit2]
        case "\"": return [C64Key.digit2, C64Key.shift]
       ...
```

**Reply by neleabels** [reply]
Thanks! That's very helpful info.
